kafka-ui-react-app/src/components/common/BytesFormatted/__tests__/BytesFormatted.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import BytesFormatted, { sizes } from '../BytesFormatted';
+
+describe('BytesFormatted', () => {
+  it('renders Bytes correctly', () => {
+    const component = shallow(
+      <BytesFormatted value={Math.floor(Math.random() * 900 + 100)} />
```

Comment:
I prefer to avoid random data in tests. Lets use const value like 533. and check it like
```
expect(component.text()).toEqual('533 Bytes');
```
From the interface `value: string | number | undefined`. Please add specs for all available cases